ubyssey/static/src/styles/modules/article/_base.scss

```diff
@@ -151,8 +151,8 @@ div.article-content {
     font-family: $font-body !important;
     font-size: 17px !important;
```

Comment:
We have a SASS helper function that automatically converts `px` to `em`. You can use it like this:

 ```sass
font-size: font-size(17) !important;
```